yolapy/resources/site.py

```diff
@@ -4,56 +4,50 @@ class SiteResourceMixin(object):
     def list_sites(self, **options):
         """Return paginated list of sites.
 
-        `yola.list_sites()`
-
-        Example response:
-
-        ```
+        >>> yola.list_sites()
         {
             'count': 999,
             'previous': None,
             'next': 'https://wl.qa.yola.net/pr/sites/?page=2',
             'results': [{...}, {...}, ...]
         }
-        ```
 
-        You can pass `page_size` and `page` as keyword arguments:
+        You can pass ``page_size`` and ``page`` as keyword arguments:
 
-        `yola.list_sites(page_size=50, page=2)`
+        >>> yola.list_sites(page_size=50, page=2)
 
         You can also pass filters and odering options as keyword arguments.
-        See <https://wl.qa.yola.net/sites/> for available options.
+        See https://wl.qa.yola.net/sites/ for available options.
         """
         return self.get(self._site_path(), params=options).json()
 
     def get_site(self, site_id):
         """Return details for a particular site.
 
-        ```
-        site = yola.get_site('site_id')
-        site['name'] # => 'My Site'
-        ```
+        >>> site = yola.get_site('site_id')
+        >>> site['name']
+        'My Site'
         """
         return self.get(self._site_path(site_id)).json()
 
     def disable_site(self, site_id):
         """Disable a site.
```

Comment:
The description reflects my entire understanding of what disabling a site actually does :(

@bearnard Do you have any insight on this?